starfish/pipeline/features/spots/detector/__init__.py

```diff
@@ -44,19 +44,5 @@ def _cli(cls, args, print_help=False):
         print('Detecting Spots ...')
```

Comment:
I adjusted any tools used by the ISS pipeline to use Codebook + IntensityTable. Because all three of the previous outputs are combined into a single IntensityTable, the outputs are simplified.